src/uk/gov/defra/ffc/GitHubStatus.groovy

```diff
@@ -52,6 +52,11 @@ class GitHubStatus implements Serializable {
     static final Description = 'Snyk test'
   }
 
+  class PactBrokerTest {
```

Comment:
I would rename this to something like `PublishPact` as it's not really running a test, it's publishing any contracts to the broker.

Reply:
Agreed.